plugins/mcreator-localization/help/fr_FR/entity/data_default_value.md

```diff
@@ -0,0 +1,2 @@
+Il s'agit de la valeur par défaut de la variable de l'entité.
+Elle peut aller de la limite négative maximale à la limite positive maximale d'un *integer* (entier).
```

Comment:
You'd need to update this help tip to match the original one (add logic and string types description here).